oaknational/Oak-Web-Application
# [PUPIL-866] Bump oak components for pupil lesson wobbly borders

## Description

Music year: 1997

This makes the oak component change for pupil lesson wobbly borders live.

[Ticket here](https://www.notion.so/oaknationalacademy/Add-wobbly-borders-to-OakLessonLayout-36c65c6f4ca34fd89c518208acb0bef6)

## How to test

1. Go to https://deploy-preview-2633--oak-web-application.netlify.thenational.academy/pupils/programmes/citizenship-secondary-year-9-l/units/how-is-the-uk-governed-5ded/lessons/how-is-the-country-run-ccvk0d/overview
2. You should see a wobbly border around the content.

## Checklist
